### PR TITLE
EST components

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds information about EST component management.

REVIEWERS - I had questions, which are inline. Please review carefully!

### CODE REVIEW

File: docs/photoniq/est/manage-components.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+5. Enter a **Component Name** and then click **Continue**.
+6. Enable permissions and then click **Continue**. Each component requires different permissions, which are described in the console.
+7. Enter any required setup information and then click **Continue**. Each component has different setup fields, which are described in the console.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - Do users need to get the component metadata before they add a new component so they know what to enter?
```
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-components.md]: Yes, when user clicks on new component 
GET API api/est/v1/components/supported-components
Is called listing summary of all the supported components

When user selects specific component to configure , then 
GET API /api/est/v1/components/supported-components/{component}
Is called , which makes all related metadata available to the user

File: docs/photoniq/est/manage-components.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+2. Click **Components**.
+3. Click the stacked dots on the line of the component that you want to view settings for, and then click **Edit**.
+4. Change any settings that you want, and then click **Update**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - Do users need to get the component settings before they add a new component so they know what to enter?
```
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-components.md]: Yes, User need to call this API to get the existing component settings https://www.macrometa.com/docs/apiEst#/paths/api-est-v1-components-settings-component/get

File: docs/photoniq/est/manage-components.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Components**.
+3. In the **Status** column, click the toggle to enable or disable the component.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - I have no idea which API call I would use for this task based on the API Reference. There is no information.
```
Comment (diana-macrometa) [inline docs/photoniq/est/manage-components.md]: @ashish-macrometa, please provide the API call.

File: docs/photoniq/est/manage-components.md (hunk 2 above)
Comment (diana-macrometa) [inline docs/photoniq/est/manage-components.md]: @ashish-macrometa, please answer this question.

File: docs/photoniq/est/manage-components.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+2. Click **Components**.
+3. Click the stacked dots on the line of the component that you want to view settings for, and then click **Delete**.
+4. Confirm your choice by clicking **Delete**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - Do users need to get the component settings so they know what name to enter for deletion?
```
Comment (diana-macrometa) [inline docs/photoniq/est/manage-components.md]: @ashish-macrometa, please answer this question.

Comment (ashish-macrometa) [inline docs/photoniq/est/manage-components.md]: User can use this API /api/est/v1/components/settings to get list of components settings, user just need to know the name of component to delete the setting 
DELETE /api/est/v1/components/settings/{component}

File: docs/photoniq/est/manage-components.md (hunk 3 above)
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-components.md]: PATCH API is used to update component settings /api/est/v1/components/settings/{component} 

https://www.macrometa.com/docs/apiEst#/paths/api-est-v1-components-settings-component/patch

File: docs/photoniq/est/manage-components.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/est/manage-components.md]: I'm not talking about the UI. If I was just using the API, what would I need to do in order to add a new component? I am guessing get the specific supported component to know what parameters to enter, then enter those values in the add component endpoint?

Comment (ashish-macrometa) [reply docs/photoniq/est/manage-components.md]: User would need to call GET API /api/est/v1/components/supported-components/{component}. to get available settings for specific component

**Review by ashish-macrometa — APPROVED** [review]

**Pull request merged**